RefactoringsExperiments/src/main/java/org/jetbrains/research/extractMethodExperiments/PluginRunner.java

```diff
@@ -62,7 +63,18 @@ private void runExtractions(CommandLine cmdLine) {
                 positiveRefactoringsExtractionRunner.run();
             }
         }
-
+        if (cmdLine.hasOption("generateNegativeSamples")) {
```

Comment:
it is an exact copy of the code below, isn't it?
see lines 78-89

Reply:
yeah, accidental misshap during tests of anticopypaster